packages/back-end/src/integrations/Redshift.ts

```diff
@@ -34,10 +35,28 @@ export default class Redshift extends SqlIntegration {
   ensureFloat(col: string): string {
     return `${col}::float`;
   }
-  getInformationSchemaFromClause(): string {
-    return "SVV_COLUMNS";
-  }
-  getInformationSchemaTableFromClause(): string {
-    return "SVV_COLUMNS";
+  generateTableName(
```

Comment:
Given that the name of this method isn't specific to the information schema anymore, I added some fallback logic after the early return.

Though, I don't think that code will be executed ever in it's current form, but it felt odd to simply return `SVV_COLUMNS` for a method with such a broad name.